build.gradle

```diff
@@ -51,6 +51,29 @@ project(":desktop") {
 
 project(":core") {
     apply plugin: "java"
+    apply plugin: 'pmd'
+    apply plugin: 'findbugs'
+
+    findbugs{
+        ignoreFailures=true
+        findbugsTest.enabled=false
+    }
+    tasks.withType(FindBugs) {
+        reports {
+            xml.enabled = false
+            html.enabled = true
+        }
+    }
+    tasks.withType(Pmd){
+        reports{
+            xml.enabled=true
+            html.enabled=true
+        }
+    }
+    pmd {
+        ignoreFailures = true
```

Comment:
Same as above

Reply:
It was to ensure the build continuing even if there were warnings during setup, it's now fixed.